Dockerfile

```diff
@@ -1,32 +1,16 @@
-FROM centos:6.7
+FROM mozillabteam/bmo-base:slim
 MAINTAINER Dylan William Hardison <dylan@mozilla.com>
 
-RUN yum update -y && \
-    yum install -y perl perl-core mod_perl httpd wget tar openssl mysql-libs gd git && \
-    wget -q https://s3.amazonaws.com/moz-devservices-bmocartons/bmo/vendor.tar.gz && \
-    tar -C /opt -zxvf /vendor.tar.gz bmo/local/ bmo/LIBS.txt bmo/cpanfile bmo/cpanfile.snapshot && \
+RUN wget -q https://s3.amazonaws.com/moz-devservices-bmocartons/bmo/vendor.tar.gz && \
```

Comment:
this is currently built by the carton_bundles repo, and will later be moved to circleci